### DISCUSSION TITLE
Rog Ally Fallout 4 Double Input

### DISCUSSION THREAD

**Discussion by thisissteven58971** [Valkirie/HandheldCompanion#815] [Q&A]
So I'd tried disabling the Steam Input and make sure there's only one device unhide. However, when I start Fallout 4 on Steam I got double clicking issue which makes navigating difficult. When I turn off HC, the issue is no longer existed. That makes me think could be contributed by double input(I could be wrong).But I really want the gyro aiming working for this game. Am I missing something that can lead to this issue? Thanks in advance for any assistance!

**Comment by digimancer** [comment]
If your "Game Controllers" screen has more than 1 gamepad listed, your [HidHide](https://github.com/nefarius/HidHide) is setup wrong. 
![image](https://github.com/Valkirie/HandheldCompanion/assets/12486400/fcbbc109-cd8f-4d36-b05a-7b15f9117111)

**Reply by thisissteven58971** [reply]
I checked my HidHide and I did check the unused controllers and made them cloaked.

**Reply by digimancer** [reply]
Did you check the Game Controllers like in my screenshot? Does it show more than one? 
If there are more than one controllers, then HidHide is wrong. If not, something else is wrong.

**Reply by digimancer** [reply]
Another thing to check:
![image](https://github.com/Valkirie/HandheldCompanion/assets/12486400/3cb05aaa-ebc1-488d-b334-8d4c735c8bd9)

**Comment by CasperH2O** [comment]
No response from original poster on follow up questions to help out.

Closing ticket.
